docs/scripts/mdx/watcher.ts

```diff
@@ -0,0 +1,27 @@
+import { generateMDX } from "./document"
+import { generateMDXMap } from "./map"
+import { initMDX } from "./package"
+import { generateMDXPaths } from "./paths"
+import type { EventName } from "./utils"
+import { getLocale, getSlug, isDefaultLocale } from "./utils"
+
+initMDX()
+
+export async function watchMDX(eventName: EventName, filePath: string) {
+  if (eventName === "addDir" || eventName === "unlinkDir") return
+
+  const locale = getLocale(filePath)
+  const slug = getSlug(filePath)
+
+  if (eventName === "add" || eventName === "change") {
+    const mdx = await generateMDX(filePath)
+
```

Comment:
We should replace backslashes with `/` for Windows.

```suggestion
  if (eventName === "add" || eventName === "change") {
    const mdx = await generateMDX(filePath.replace(/\\/g, "/"))

```

Reply:
Fixed